src/main/java/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import service.ReaderService;
+
+public class ReaderServiceImpl implements ReaderService {
+    private final List<String> readLines;
+
+    public ReaderServiceImpl() {
+        this.readLines = new ArrayList<>();
+    }
+
+    @Override
+    public void readDataFromFile(File file) {

```

Comment:
```suggestion
    @Override
    public List<String> readFile(String filePath) {
```
Don’t complicate your solution. This method can return List<String> and you don't need to have a field and method getReadLines